diff --git a/lib/ip.js b/lib/ip.js
index c1799a8c50f42744320a529c3a9e2b77bac74898..e6c83dfb9ce4c59e74f0dd8549900ba54f0bb3f6 100644
--- a/lib/ip.js
+++ b/lib/ip.js
@@ -95,8 +95,11 @@ ip.isV4Format = function(ip) {
 ip.isV6Format = function(ip) {
   return ipv6Regex.test(ip);
 };
+function _resolveFamily(family) {
+  return typeof family === 'number' ? 'ipv' + family : family.toLowerCase();
+}
 function _normalizeFamily(family) {
-  return family ? family.toLowerCase() : 'ipv4';
+  return family ? _resolveFamily(family) : 'ipv4';
 }
 
 ip.fromPrefixLen = function(prefixlen, family) {
@@ -368,7 +371,7 @@ ip.address = function(name, family) {
   //
   if (name && name !== 'private' && name !== 'public') {
     var res = interfaces[name].filter(function(details) {
-      var itemFamily = details.family.toLowerCase();
+      var itemFamily = _resolveFamily(details.family);
       return itemFamily === family;
     });
     if (res.length === 0)
@@ -382,7 +385,7 @@ ip.address = function(name, family) {
     // when this is called.
     //
     var addresses = interfaces[nic].filter(function (details) {
-      details.family = details.family.toLowerCase();
+      details.family = _resolveFamily(details.family);
       if (details.family !== family || ip.isLoopback(details.address)) {
         return false;
       } else if (!name) {
